cl/disclosures/api_serializers.py

```diff
@@ -88,13 +97,20 @@ class FinancialDisclosureSerializer(
     positions = PositionSerializer(many=True, read_only=True)
     reimbursements = ReimbursementSerializer(many=True, read_only=True)
     spouse_incomes = SpouseIncomeSerializer(many=True, read_only=True)
-    person = serializers.HyperlinkedRelatedField(
-        many=False,
-        view_name="person-detail",
-        queryset=Person.objects.all(),
-        style={"base_template": "input.html"},
-    )
+    person = JudgeSerializer(many=False, read_only=True)
 
     class Meta:
         model = FinancialDisclosure
         exclude = ("download_filepath",)
+
+    def to_representation(self, data):
+        data = super(FinancialDisclosureSerializer, self).to_representation(
+            data
+        )
+        data[
+            "filepath"
+        ] = f"https://{settings.AWS_S3_CUSTOM_DOMAIN}/{data['filepath']}"
+        data[
+            "thumbnail"
+        ] = f"https://{settings.AWS_S3_CUSTOM_DOMAIN}/{data['thumbnail']}"
+        return data
```

Comment:
This looks sorta OK, but did you see this: https://stackoverflow.com/a/35522896/64911

Reply:
yeah and we have some examples in CL I think - I just thought the stack I found was cleanest.

Reply:
I changed it - thinking that the full example was too much - but I changed it to reflect the stack overflow answer.